src/app/modules/submission/multirespondent-submission/multirespondent-submission.controller.ts

```diff
@@ -347,6 +377,115 @@ const runMultirespondentWorkflow = ({
   )
 }
 
+const sendMrfCompletionEmailIfWorkflowCompleted = ({
```

Comment:
This function could likely be moved into .service as it doesn't look like it's dealing with HttpRequests/Responses